scripts/build-bpf.sh

```diff
@@ -23,15 +23,13 @@ cargo test --locked
 
 # Re-run tests affected by features
 cargo test --locked --features pythnet -- \
+      test_add_price \
+      test_add_price_pythnet_fails_to_add_v1 \
+      test_add_publisher \
```

Comment:
adding tests manually here is going to be very error prone -- people will forget to do this all the time. is it much more expensive to rerun everything?

Reply:
Okay, maybe it's fine, the difference is ca. 1min and some change